bound/kt/src/test/kotlin/web5/sdk/dids/BearerDidTest.kt

```diff
@@ -1,41 +1,41 @@
 package web5.sdk.dids
 
-import org.junit.jupiter.api.Assertions.assertEquals
-import org.junit.jupiter.api.Assertions.assertNotNull
-import org.junit.jupiter.api.Test
-import web5.sdk.crypto.keys.InMemoryKeyManager
-import web5.sdk.crypto.keys.Jwk
-import web5.sdk.dids.methods.jwk.DidJwk
-import web5.sdk.rust.ed25519GeneratorGenerate
+//import org.junit.jupiter.api.Assertions.assertEquals
+//import org.junit.jupiter.api.Assertions.assertNotNull
+//import org.junit.jupiter.api.Test
+//import web5.sdk.crypto.keys.InMemoryKeyManager
+//import web5.sdk.crypto.keys.Jwk
+//import web5.sdk.dids.methods.jwk.DidJwk
+//import web5.sdk.rust.ed25519GeneratorGenerate
```

Comment:
remove commented imports?

Reply:
oh yeah, I meant to comment on this, I'm temporarily removing this until I get around to bearer DIDs, I can go ahead and remove tho as when I come around it'll be all new